kafka/src/test/java/org/axonframework/extensions/kafka/eventhandling/consumer/streamable/AbstractKafkaTrackingTokenSerializeTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2010-2019. Axon Framework
```

Comment:
The notice should stretch to this year:

```suggestion
 * Copyright (c) 2010-2021. Axon Framework
```